bank_statement_parse/README.rst

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
README.rst must be formatted as https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst